### PR TITLE
Remove unused files and code; refactor

### PR DESCRIPTION
**Pull request by yannikschaelte**

### CODE REVIEW

File: .github/workflows/ci_pr.yml [hunk 1]
```diff
@@ -24,6 +24,14 @@ jobs:
...
           sudo apt-get install \
             swig3.0 libatlas-base-dev libhdf5-serial-dev
+      
+      - name: Download COPASI
+        run: |
+          COPASI_VERSION="4.29.228-Linux-64bit"
+          wget "https://github.com/copasi/COPASI/releases/download/Build-228/COPASI-${COPASI_VERSION}.tar.gz"
+          tar -xzvf COPASI-${COPASI_VERSION}.tar.gz
+          mv COPASI-${COPASI_VERSION} COPASI
+          export COPASI_DIR_BIN=COPASI/bin
```
Comment (paulstapor) [inline .github/workflows/ci_pr.yml:34]: wonderful, many thanks!

File: Python_Scripts/C.py [hunk 2]
```diff
@@ -27,6 +27,10 @@
 elif DIR_BASE == 'TEST':
     DIR_BASE = DIR_TEST_BASE
 
+# Folder with COPASI binaries, user-dependent, to reproduce LSODA simulations
+DIR_COPASI_BIN = os.getenv('COPASI_DIR_BIN', 'COPASI/bin')
+
+
```
Comment (paulstapor) [inline Python_Scripts/C.py:33]: good idea to make it an environmental variable

File: Python_Scripts/C.py [hunk 3]
```diff
@@ -41,15 +45,6 @@
...
 
-###############################################################################
-# Data directories
-
-# Simulation data base folder
-DIR_DATA = os.path.join(DIR_BASE, 'Data')
-# Folder for most of the sub-studies
-DIR_DATA_WHOLESTUDY = os.path.join(DIR_DATA, 'WholeStudy')
-# Folder for the tolerances sub-study
-DIR_DATA_TOLERANCES = os.path.join(DIR_DATA, 'TolerancesStudy')
```
Comment (paulstapor) [inline Python_Scripts/C.py:52]: yes, well... I didn't take the time to look for all the things that can be cleaned up. Thanks for taking care of that!

**Review by paulstapor — APPROVED** [review]
This is a lot of red lines... Cool!

File: Python_Scripts/C.py (hunk 3 above)
Comment (yannikschaelte) [reply Python_Scripts/C.py:52]: yes, basically you renamed "Data" to "results", which also makes more sense, so I updated that. all paths in plots yet to be updated, but those need extra care anyway.

Comment (paulstapor) [reply Python_Scripts/C.py:52]: agreed, but this "extra care" makes sense, as now, the naming schemes for the result files are more indicative...
and overall, less data is saved, but more information...

**Pull request merged**